eZ/Publish/Core/Persistence/Cache/Tests/AbstractCacheHandlerTest.php

```diff
@@ -27,14 +27,16 @@ abstract class AbstractCacheHandlerTest extends AbstractBaseHandlerTest
      * @param string $method
      * @param array $arguments
      * @param array|null $tags
-     * @param string|null $key
+     * @param string|array|null $key
      * @param mixed $returnValue
      */
-    final public function testUnCachedMethods(string $method, array $arguments, array $tags = null, string $key = null, $returnValue = null)
+    final public function testUnCachedMethods(string $method, array $arguments, array $tags = null, $key = null, $returnValue = null)
```

Comment:
I don't really like this, but you know it ;)
Not very important though, especially that my review is very late.

Reply:
I know, but it's tests, and it's alignment of the two abstract test classes